### PR TITLE
feat: added notifications tests

### PR DESCRIPTION
**Pull request by DarthXerox**

### CODE REVIEW

File: PV260_Project/Tests/BLTests/NotificationsTests.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+using HtmlAgilityPack;
+
+using System.Linq;
+namespace Tests.BLTests;
+
+[TestFixture]
+public class NotificationsTests
+{
+    private List<HoldingChanges> _changes = new List<HoldingChanges>();
+    private CultureInfo _culture;
```
Comment (MatejMichalek) [inline PV260_Project/Tests/BLTests/NotificationsTests.cs:20]: _culture is assigned but never used --> can be removed

File: PV260_Project/Tests/BLTests/NotificationsTests.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+
+    private string _holdingsStr =
```
Comment (MatejMichalek) [inline PV260_Project/Tests/BLTests/NotificationsTests.cs:23]: never used, also can be removed

File: PV260_Project/Tests/BLTests/NotificationsTests.cs [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+using Microsoft.Extensions.Options;
+using HtmlAgilityPack;
+
+using System.Linq;
+namespace Tests.BLTests;
+
+[TestFixture]
+public class NotificationsTests
+{
+    private List<HoldingChanges> _changes = new List<HoldingChanges>();
```
Comment (MatejMichalek) [inline PV260_Project/Tests/BLTests/NotificationsTests.cs:19]: IDE offers that it is redundant type specification .. but I think it's ok
same in _mailSettings

**Review by MatejMichalek — COMMENTED** [review]
I was just nitpicking, nothing serious
Good idea with TestUtils 👍

**Review by Bevilton — APPROVED** [review]

**Pull request merged**